src/setup/templates/rgw.logrotate.tmpl

```diff
@@ -0,0 +1,13 @@
+TMP_LOG_PATH/*.log
+{
+    rotate 7
+    maxsize 50M
+    weekly
+    compress
+    missingok
+    copytruncate
+    postrotate
+        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x "ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror" || true
```

Comment:
do we need these sections ? 
1) postrotate  - no action since rgw is running inside the container .so explicit killing rgw instance might cause container restarts.
2) ' su root cep ' this line is also not required.

Reply:
Yeah why this kill action?? We dont want to restart rgw or dependencies per log rotate operation! Why its been done here or even if default at all ?

Reply:
Ack.
Removed the postrotate script.
It was the default ceph configuration

Reply:
> It was the default ceph configuration

Try to gather the reason. If we dont do it, does it have any implication